python/setup_ocnice_config.py

```diff
@@ -81,14 +86,16 @@ def check_env(envVarName):
                   '{}/mapping'.format(scratchDir))
 add_config_option(config, 'output', 'mpasClimatologySubdirectory',
                   '{}/clim/mpas/'.format(scratchDir))
-add_config_option(config, 'output', 'mpasRegriddedClimSubdirectory',
-                  '{}/clim/mpas/regridded'.format(scratchDir))
 add_config_option(config, 'output', 'timeSeriesSubdirectory',
-                  '{}/timeSeries/'.format(scratchDir))
+                  '{}/timeseries/'.format(scratchDir))
+add_config_option(config, 'output', 'timeCacheSubdirectory',
+                  '{}/timecache/'.format(scratchDir))
```

Comment:
Oops, this is a rather old and unused config option.  you can (and should) delete these two lines.  I'll do an MPAS-Analysis PR to remove `timeCacheSubdirectory`

Reply:
will do. I have to say, I wasn't sure where it was relevant, but I decided to err on the side of caution.